python/vyos/template.py

```diff
@@ -341,19 +341,21 @@ def get_dhcp_router(interface):
     """ Static routes can point to a router received by a DHCP reply. This
     helper is used to get the current default router from the DHCP reply.
 
-    Returns False of no router is found, returns the IP address as string if
+    Returns None if no router is found, returns the IP address as string if
     a router is found.
     """
-    interface = interface.replace('.', '_')
-    lease_file = f'/var/lib/dhcp/dhclient_{interface}.leases'
+    lease_file = f'/var/lib/dhcp/dhclient_{interface}.lease'
```

Comment:
Not a problem but the `/var/lib/dhcp/dhclient_X.lease` file is abandoned in new implementations and used only in old scripts. I would prefer to not reuse it again.